packages/wrangler/src/hyperdrive/create.ts

```diff
@@ -35,6 +36,10 @@ export async function handler(
 	});
 	logger.log(
 		`✅ Created new Hyperdrive config\n`,
-		JSON.stringify(database, null, 2)
+		TOML.stringify({
```

Comment:
This should be using the new `formatConfigSnippet()` helper introduced in https://github.com/cloudflare/workers-sdk/pull/7230